samcli/commands/check/lib/command_context.py

```diff
@@ -0,0 +1,79 @@
+import os
+import functools
+
+import click
+import boto3
+
+from samtranslator.translator.managed_policy_translator import ManagedPolicyLoader
+from samtranslator.translator.translator import Translator
+from samtranslator.public.exceptions import InvalidDocumentException
+from samtranslator.parser import parser
+from boto3.session import Session
+
+from samcli.commands.local.cli_common.user_exceptions import SamTemplateNotFoundException
+from samcli.yamlhelper import yaml_parse
+
+from samcli.lib.replace_uri.replace_uri import _replace_local_codeuri as external_replace_local_codeuri
+from ..exceptions import InvalidSamDocumentException
+
+
+class CheckContext:
+    def __init__(self, ctx, template_path):
```

Comment:
1. Please add typing to the variables.
2. If that `ctx` is coming from `click` library, please keep it outside of this class but pass attributes which you will be using in this one

Reply:
1. Do you mean in the parameters themselves (i.e. ctx: str, template_path: str), or in the documentation for the class like here: "https://github.com/aws/aws-sam-cli/blob/59c85768356089edb265c2ea7f53bce2412f9e19/samcli/commands/deploy/deploy_context.py#L190"?
2. Ok.